Experiments/Experiments/FeatureFlag.swift

```diff
@@ -81,4 +81,39 @@ public enum FeatureFlag: Int {
     /// Enables signing up for a WP.com account.
     ///
     case wpcomSignup
+
+    // MARK: - Performance Monitoring
+    //
+    // This flags are not transient. That is, they are not here to help us rollout a feature,
+    // but to serve a safety switches to granularly turn off performance monitoring if it looks
+    // like we are consuming too many events.
```

Comment:
This will make more sense once the feature toggles will be configurable remotely 😅